lib/actions/file.js

```diff
@@ -41,7 +44,10 @@ exports.listFileNames = function(b2, args) {
         body: JSON.stringify({
             bucketId: bucketId,
             startFileName: startFileName ? startFileName : '',
-            maxFileCount: maxFileCount ? maxFileCount : 100
+            maxFileCount: maxFileCount ? maxFileCount : 100,
+            prefix: prefix ? prefix : '',
+            delimiter: delimiter ? delimiter : null
```

Comment:
@yakovkhalinsky Those are the default values that the B2 API expects (see documentation). We were working on a project today that includes/needs these modifications and ran into an issue with `delimiter = ''`. The API response was pretty clear that, if included, `delimiter` must be a string with a length of 1. The same rules don't apply with `prefix` as it defaults to an empty string anyway.